### PR TITLE
Feat: migrate v-img to img

### PR DESCRIPTION
**Pull request by tewshi**
## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/helper/display/icons/ChainIcon.vue [hunk 1]
```diff
@@ -13,12 +13,14 @@ const props = withDefaults(defineProps<Props>(), {
 const { chain } = toRefs(props);
 
 const { getChainImageUrl } = useSupportedChains();
+
+const src = computed(() => get(getChainImageUrl(chain)));
```
Comment (lukicenturi) [inline frontend/app/src/components/helper/display/icons/ChainIcon.vue:17]: `const src = getChainImageUrl(chain)` is enough right?

Comment (tewshi) [reply frontend/app/src/components/helper/display/icons/ChainIcon.vue:17]: oh, should be since the function returns a computed value.

Comment (tewshi) [reply frontend/app/src/components/helper/display/icons/ChainIcon.vue]: done

**Comment by kelsos** [comment]
@tewshi PR can you rebase and squash on top of develop? there are also some build errors when rebased to fix.

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**